components/Button.js

```diff
@@ -47,16 +56,24 @@ export function Button({
 			<StyledLink href={href ? href : null} asChild>
 				<StyledTouchableHighlight
 					activeOpacity={0.6}
-					underlayColor="#DDDDDD"
+					// underlayColor="#DDDDDD"
 					onPress={() => press()}
-					className={`flex h-[50px] items-center justify-center rounded-full bg-offwhite border border-[${borderClr}]
-				${width ? width : "w-11/12"}
-			`}
+					className={`flex items-center justify-center rounded-full 
+						${height ? height : "h-[50px]"} 
+						${width ? width : "w-11/12"}
+					`}
+					style={{ 
+						backgroundColor: bgColor,
+						borderWidth: 1,
+						borderColor: borderClr
+					}}
 				>
 					<StyledText
-						className={`font-bold text-[${txtColor}] ${
-							textSize ? textSize : "text-[20px]"
-						} ${textStyles}`}
+						className={`font-bold 
+							${textSize ? textSize : "text-[20px]"}
+							${textStyles}
+						`}
+						style={{ color: txtColor }}
```

Comment:
should be added as in nativewind

Reply:
If you can, feel free to. @dev-arekusandoru